docs/Admin/Appearance/Themes.md

```diff
@@ -1,83 +1,86 @@
-7.  Log in to your Omeka admin panel
-8.  Navigate to Appearance in the top nav bar. You should immediately be in the Themes section.
-    - If the theme is not visible, return to step 6 and double-check that the folder is in the right location ('/themes') and that the folder name for the theme does not start with 'theme-'
-9.  Click "Use This Theme" to finish the process.
-    - Click “Configure Theme” if desired.
+Watch the [screencast](https://vimeo.com/153819886){target=_blank} demonstrating the process of installing plugins and themes (with close-captioning).
+
+To add and activate a new theme for your Omeka installation, follow these steps:
+
+1. [Download and unzip the compressed theme file](https://omeka.org/classic/themes/){target=_blank} on your computer.
+1. Open your FTP client and use your FTP login info to access your site. Or, if you are using cPanel File Manager, WebFTP, or another in-browser tool, log in there. 
+1. Navigate to your Omeka folder (it should have the same name as your Omeka install), open it, then open the `/themes` folder.
+1. Locate the theme folder on your computer and upload it into your Omeka website's `/themes` folder. 
+    - Sometimes the theme unzips into a folder of the same name, so make sure to go inside it and upload the folder that only contains the theme name, not the version number. For example, you may have downloaded the `theme-minimalist-v2.5.2.zip` file and extracted it to a `theme-minimalist-v2.5.2` folder, which contains a `minimalist` folder. Upload the `minimalist` folder. 
+    - You may also be able, or required, to upload the zipped file and unzip it here, if your file client does not allow you to upload a folder. Remember to watch for an extra folder level in this case.
+1. In another browser window, log in to your Omeka admin panel (`https://youromekasite.org/admin`).
+1.  Navigate to the Appearance tab in the upper navigation of the admin dashboard, which will display the Themes tab first, and scroll down to find the theme you uploaded. Themes are listed newest first.
+    - If the theme is not visible, return to the 4th step and double-check that the folder is in the right location ('/themes') and that the folder name for the theme does not start with 'theme-'.
+1. Click “Use this theme” under the theme thumbnail to finish the process.
+1. Click “Configure theme” after the page reloads. Read the documentation for your chosen theme to understand what configuration options are available.
 
 Selecting a Theme
 -------------------------------------------------------------
-You can browse the themes you have installed for your Omeka Classic site by going to the Appearance tab in the upper admin navigation. The Themes tab in the Appearance section will be the first which loads. 
+You can browse the themes you have installed for your Omeka Classic site by going to the Appearance tab in the upper admin navigation. The Themes tab in the Appearance section will be the first to display. 
```

Comment:
Just testing to see if this works how I remember still....

Reply:
This one was made as an "add single comment" in the web interface (the plus icons on the lines when viewing changes), and it's showing up as I generally expect these to.